README.md

```diff
@@ -13,111 +13,84 @@ This plugin intends to support linting of ES2015+ (ES6+) import/export syntax, a
 
 ## Rules
 
-### Static analysis
-
-* Ensure imports point to a file/module that can be resolved. ([`no-unresolved`])
-* Ensure named imports correspond to a named export in the remote file. ([`named`])
-* Ensure a default export is present, given a default import. ([`default`])
-* Ensure imported namespaces contain dereferenced properties as they are dereferenced. ([`namespace`])
-* Restrict which files can be imported in a given folder ([`no-restricted-paths`])
-* Forbid import of modules using absolute paths ([`no-absolute-path`])
-* Forbid `require()` calls with expressions ([`no-dynamic-require`])
-* Prevent importing the submodules of other modules ([`no-internal-modules`])
-* Forbid webpack loader syntax in imports ([`no-webpack-loader-syntax`])
-* Forbid a module from importing itself ([`no-self-import`])
-* Forbid a module from importing a module with a dependency path back to itself ([`no-cycle`])
-* Prevent unnecessary path segments in import and require statements ([`no-useless-path-segments`])
-* Forbid importing modules from parent directories ([`no-relative-parent-imports`])
-* Prevent importing packages through relative paths ([`no-relative-packages`])
-
-[`no-unresolved`]: ./docs/rules/no-unresolved.md
-[`named`]: ./docs/rules/named.md
-[`default`]: ./docs/rules/default.md
-[`namespace`]: ./docs/rules/namespace.md
-[`no-restricted-paths`]: ./docs/rules/no-restricted-paths.md
-[`no-absolute-path`]: ./docs/rules/no-absolute-path.md
-[`no-dynamic-require`]: ./docs/rules/no-dynamic-require.md
-[`no-internal-modules`]: ./docs/rules/no-internal-modules.md
-[`no-webpack-loader-syntax`]: ./docs/rules/no-webpack-loader-syntax.md
-[`no-self-import`]: ./docs/rules/no-self-import.md
-[`no-cycle`]: ./docs/rules/no-cycle.md
-[`no-useless-path-segments`]: ./docs/rules/no-useless-path-segments.md
-[`no-relative-parent-imports`]: ./docs/rules/no-relative-parent-imports.md
-[`no-relative-packages`]: ./docs/rules/no-relative-packages.md
+<!-- begin auto-generated rules list -->
+
+💼 Configurations enabled in.\
+☑️ Enabled in the `recommended` configuration.\
+☑️<sup>⚠️</sup> Warns in the `recommended` configuration.\
+❗ Enabled in the `errors` configuration.\
+⚠️ Warns in the `warnings` configuration.\
```

Comment:
it's a little confusing to have the same emoji for the "warnings" config as for a modifier on another config to indicate that it warns.

Reply:
How about using a new emoji for the `warnings` config via `--config-emoji warnings,🚸`. I pushed the update.


### Before:

💼 Configurations enabled in.\
☑️ Enabled in the `recommended` configuration.\
☑️<sup>⚠️</sup> Warns in the `recommended` configuration.\
❗ Enabled in the `errors` configuration.\
⚠️ Warns in the `warnings` configuration.\
⌨️<sup>🚫</sup> Disabled in the `typescript` configuration.

### After:

💼 Configurations enabled in.\
☑️ Enabled in the `recommended` configuration.\
☑️<sup>⚠️</sup> Warns in the `recommended` configuration.\
❗ Enabled in the `errors` configuration.\
🚸<sup>⚠️</sup> Warns in the `warnings` configuration.\
⌨️<sup>🚫</sup> Disabled in the `typescript` configuration.

Reply:
Looks great!